testing/scenario_app/ios/Scenarios/Scenarios/Assets.xcassets/Contents.json

```diff
@@ -0,0 +1,6 @@
+{
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
+  }
+}
```

Comment:
Do we need newlines here?

Reply:
These are generated files. I could add a newline though